concent_api/core/payments/bankster.py

```diff
@@ -238,46 +201,47 @@ def finalize_payment(deposit_claim: DepositClaim) -> Optional[str]:
         elif available_funds_without_claims < deposit_claim.amount:
             deposit_claim.amount = available_funds_without_claims
 
-        # If the DepositClaim still exists at this point, Bankster uses SCI to create an Ethereum transaction.
```

Comment:
I think it was about keeping `DepositAccount` for the time SCI calls below. But maybe it changed in meantime. @cameel can you verify?

Reply:
See https://github.com/golemfactory/concent/pull/1041#discussion_r242938574